pyproject.toml

```diff
@@ -1,5 +1,6 @@
 [build-system]
 requires = [
+    "frequenz-api-common == 0.2.0",
```

Comment:
I don't think this is required as a build dependency. For building we are using the submodule.

Reply:
The tests do pass without this line. Removed. :rocket: